client/boinc_cmd.cpp

```diff
@@ -188,6 +196,167 @@ void show_alerts(RPC_CLIENT &rpc) {
     }
 }
 
+////////////// --get_task_summary start ////////////////
+
+typedef vector<const char*> STR_LIST;
+
+// given: a list of lines, each consisting of n columns
+// for each column: find the longest entry.
+// Then display the lines so the columns line up.
+//
+void show_str_lists(vector<STR_LIST> lines, int ncols) {
```

Comment:
I believe `ncols` will always be greater than 0. I suggest changing it's type to `size_t`